ios/RNCallKeep/RNCallKeep.m

```diff
@@ -118,10 +121,10 @@ - (void)sendEventWithNameWrapper:(NSString *)name body:(id)body {
     if (_hasListeners) {
         [self sendEventWithName:name body:body];
     } else {
-        NSDictionary *dictionary = @{
-            @"name": name,
-            @"data": body
-        };
+        NSDictionary *dictionary = [NSDictionary dictionaryWithObjectsAndKeys:
+                              name, @"name",
+                              body, @"data",
+                              nil];
```

Comment:
Certain events have `data` as null - this approach gets rid of an exception when that's the case.